testsuites/CBLTester/CBL_Functional_tests/APITests/test_scopesCollections.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+import random
+import math
+from keywords.utils import random_string
+
+
+@pytest.mark.usefixtures("class_init")
+class TestScopeCollection(object):
+
+    @pytest.mark.parametrize("db_name, scope, collection", [
+        (random_string(6), "{} {}".format(random_string(3), random_string(3)), "{} {}".format(random_string(3), random_string(3))),
+        (random_string(6), random_string(6), "{} {}".format(random_string(3), random_string(3))),
+        (random_string(6), "{} {}".format(random_string(3), random_string(3)), random_string(6)),
+        (random_string(6), random_string(6), random_string(6))
+    ])
+    def test_scope_collection_name_with_space(self, db_name, scope, collection):
+        '''
+         @summary: Creating scope and collections with space in names
+        '''
+        db = self.db_obj.create(db_name)
+        try:
+            self.db_obj.createCollection(db, collection, scope)
+            created_Collection = self.db_obj.collectionObject(db, collection, scope)
+            created_CollectionName = self.collection_obj.collectionName(created_Collection)
+            assert created_CollectionName == collection, "Scope and collection created but not found"
+        except:
+            print("Invalid scope and collection name")
```

Comment:
Maybe add the invalid name that's failing? That way it'll be easier to get the relevant string just from reading the logs.